iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -46,6 +46,49 @@ static void getMatmulConfig(SmallVectorImpl<TileWorkgroupSizePair> &tileSizes) {
   tileSizes.push_back(TileWorkgroupSizePair({{1, 128, 8}, {32, 1, 1}}));
 }
 
+/// Return the best combination of tile size and wg size when using tensorcore
+/// operations.
+static void getTensorCoreConfig(
+    SmallVectorImpl<TileWorkgroupSizePair> &tileSizes) {
+  tileSizes.push_back(TileWorkgroupSizePair({{64, 64, 16}, {64, 2, 1}}));
```

Comment:
I think  `tileSizes.emplace_back({{..}, {..}})` also works?

Reply:
true

Reply:
> true

Actually can't get the compiler to take this. Somehow it is not able to infer the right type for the std::array when trying to do that.